queue_entry.py

```diff
@@ -553,7 +552,7 @@ def execute(self):
             # Create a centred postions of the current postion
             pos_dict = self.diffractometer_hwobj.getPositions()
             cpos = queue_model_objects.CentredPosition(pos_dict)
-            pos = shape_history.Point(None, cpos, None) #, True)
+            #pos = shape_history.Point(None, cpos, None) #, True)
```

Comment:
Hi,
Yes, all that block (559 - 571) can be commented. As it looks the idea was to assign new centred position to all tasks in the task group where the centring task is. In principle it is enought to center before all tasks and thats it. I dont see any reason to assign this point to tasks. but we can discuss this...